pkg/storage/replica.go

```diff
@@ -252,11 +252,15 @@ type Replica struct {
 	RangeID roachpb.RangeID // Should only be set by the constructor.
 
 	store *Store
-	// sha1 hash of the system config @ last gossip. No synchronized access;
-	// must only be accessed from maybeGossipSystemConfig (which in turn is
-	// only called from the Raft-processing goroutine).
-	systemDBHash []byte
-	abortCache   *AbortCache // Avoids anomalous reads after abort
+
+	// sha1 hash of the system config @ last gossip. Must be accessed only
+	// from the Raft-processing goroutine.
```

Comment:
There is no `Raft-processing goroutine` anymore. What's the real locking story here?